boards/arm/mimxrt1062_fmurt6/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright 2023 NXP
+#
+
+if (CONFIG_DISPLAY)
+message(WARNING "
```

Comment:
This is a message specific to NXP's RT1050 and 106x EVKs, because these boards may cause damage to their display circuit if it is enabled when no display is attached. Is this something you need for the FMURT6 board?

Reply:
Thanks Daniel.. This is indeed not required for FMURT6.. hence removing it..

Reply:
This has in fact not been removed: https://github.com/zephyrproject-rtos/zephyr/blob/5781478fd5c466d0fdab82af96580026b099c24e/boards/arm/mimxrt1062_fmurt6/CMakeLists.txt#L7-L11